### PR TITLE
Starting basic work for adding PCL targets for fuzzing

### PR DESCRIPTION
**Pull request by kunaltyagi**
Needs a lot of feedback on the process.

### CODE REVIEW

**Comment by kunaltyagi** [comment]
@googlebot I signed it!

**Comment by Dor1s** [comment]
@kunaltyagi any plans to finish this? Also, please check out Travis presubmit failures

**Comment by kunaltyagi** [comment]
I'll add the Google license in pcl files.

How do I go about using the existing binary files generated by PCL for io instead of writing a custom binary?

**Comment by Dor1s** [comment]
> How do I go about using the existing binary files generated by PCL for io instead of writing a custom binary?

What binary are you talking about? You need to write a fuzz target: https://github.com/google/fuzzing/blob/master/tutorial/libFuzzerTutorial.md#hello-world-fuzzer

File: projects/pcl/build.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+cmake \
+  -DCMAKE_C_COMPILER=${CC} \
+  -DCMAKE_CXX_COMPILER=${CXX} \
+  -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+  -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=${OUT} \
+  -DLIB_FUZZING_ENGINE:string={LIB_FUZZING_ENGINE} \
+  $@ \
+  ${SRC}
+
+make -j${nproc}
```
Comment (kunaltyagi) [inline projects/pcl/build.sh:28]:
```suggestion
make -j$(nproc)
```

**Pull request closed without merging**